pkgs/applications/networking/termius/default.nix

```diff
@@ -12,11 +12,12 @@
 
 stdenv.mkDerivation rec {
   pname = "termius";
-  version = "7.1.0";
+  version = "7.15.1";
 
   src = fetchurl {
-    url = "https://deb.termius.com/pool/main/t/termius-app/termius-app_${version}_amd64.deb";
```

Comment:
It's `https://deb.termius.com/pool/main/t/termius-app/termius-app_7.16.0_amd64.deb` now.
See `wget -qO- https://deb.termius.com/dists/squeeze/main/binary-amd64/Packages`.

Reply:
Thanks! i did not know that command